DDDEastAnglia.Tests/DateExtensions.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+
+namespace DDDEastAnglia.Tests
+{
+    public static class DateExtensions
+    {
+        public static DateTime January(this int day, int year)
```

Comment:
Instead of this class, we could pull in [Humanizer](https://github.com/MehdiK/Humanizer) via NuGet.